vllm/worker/habana_model_runner.py

```diff
@@ -1503,4 +1503,5 @@ def shutdown_inc(self):
             print('inc shutdown')
 
     def __del__(self):
+        self.finish_measurements()
```

Comment:
@MrGeva 
I see the in `HabanaModelRunner` the `finalize_clibration (== self.finish_measurements)` is called at `shutdown_inc` (above method)
Why then did we need to move `finish_measurements` to the DTOR?